### ISSUE TITLE
Where are the PHP communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/php#204]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is PHP supported by one or more large organizations?
* Does PHP have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about PHP? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for PHP? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in PHP? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in PHP?
* Are there popular newsletters for PHP?
* Is PHP taught at programming bootcamps? (If so, what are some examples?)
* Is PHP taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about PHP?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by kzap** [comment]
- Is PHP supported by one or more large organizations?
    - [Automatic](https://automattic.com/) (Wordpress)
    - [Symfony](https://symfony.com/) Widely used framework
    - [Laravel](https://laravel.com/) Modern framework like Rails
    - Etsy (Where Ramus Lerdorf works)
    - Zend
    - Facebook
    - Drupal
    - [PHP Framework Interop Group](https://www.php-fig.org/) (standards org)

- Does PHP have an official community manager?
   - I don't think so, [Rasmus Lerdorf](https://twitter.com/rasmus), the creator of PHP would be the closest person as he advocates for it and goes around conferences speaking about it. Each PHP project has their own community and managers, each bring their own views and opinions to the project

- Do you know of specific communities (online or offline) that are enthusiastic about PHP? (Chat communities, forums, meetups, student clubs, etc)
    - Laravel, Symfony, Wordpress, Drupal and other projects and frameworks each have their own communities, blogs, meetups, conferences, resources. It's quite a wide ecosystem.

- Are there popular conferences for PHP? (If so, what are some examples?)
    - Here are a lot: http://php.net/conferences/
    - Each framework has their own conferences and there are lots of smaller conferences also.

That's the best of what I can answer. I would say the community is very broad and wide and found in all countries because PHP has been around since the beginnings of the browser and the adoption of the internet.

To find people for PHP I suggest going to the main forums of each large framework/project like Laravel and Symfony and soliciting help there.

**Comment by kytrinyx** [comment]
This is great, @kzap thank you!

**Comment by yisraeldov** [comment]
If the goal is good quality code, I would recommend against going to some of the forums.  Most of the laravel users are very beginner, and there are things in laravel that encourage poor code quality.
